javabridge/wrappers.py

```diff
@@ -158,6 +158,12 @@ def __repr__(self):
     
     def __str__(self):
         return J.to_string(self.o)
+    
+    def __int__(self):
+        return self.intValue()
```

Comment:
I think here, you might want to throw ValueError if the class does not have an intValue method. I am guessing that

    x = J.JClassWrapper('java.util.ArrayList')()
    print x == 5

throws an exception instead of printing "False"